Makefile

```
@@ -200,6 +202,10 @@ proto: ## Build proto (Protocol Buffers) files
 	$(PROTOC) --proto_path=pkg/graveler/committed --go_out=pkg/graveler/committed --go_opt=paths=source_relative committed.proto
 	$(PROTOC) --proto_path=pkg/graveler --go_out=pkg/graveler --go_opt=paths=source_relative graveler.proto
 
+publish-scala: ## sbt publish jars to nexus and s3 bucket
+	cd clients/spark && sbt publish
```

```suggestion
	cd clients/spark && sbt publishAll
```
Or am I missing something?

---

Good catch!